tensorflow/lite/CMakeLists.txt

```diff
@@ -218,6 +218,9 @@ endif()
 if(NOT "${CMAKE_SYSTEM_NAME}" STREQUAL "Android")
   list(FILTER TFLITE_SRCS EXCLUDE REGEX ".*minimal_logging_android\\.cc$")
 endif()
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Android")
```

Comment:
It's bit confusing.
How about excluding ".*minimal_logging.*" first and add logging file per each platforms (Android, iOS, default)?

Reply:
Done